app/models/comment.rb

```diff
@@ -60,4 +60,10 @@ def self.search(
         .offset((page - 1) * Constants::SEARCH_LIMIT)
     end
   end
+
+  def self.mark_as_seen_by(user)
```

Comment:
Is there any way to prevent calling this on the model itself and limit it only to the collection?
We would really like to avoid somebody calling
```Comment.mark_as_seen_by(current_user)```

Reply:
That 'somebody' can also call Comment.delete_all